ycmd/tests/test_utils.py

```diff
@@ -105,3 +105,26 @@ def StopGoCodeServer( app ):
                  BuildRequest( completer_target = 'filetype_default',
                                command_arguments = ['StopServer'],
                                filetype = 'go' ) )
+
+
+# Prints an object in a way to show its structure and content. Useful when
+# debugging complex responses (such as the diagnostic response) in tests.
+def PrettyPrint( d, indent=0 ):
```

Comment:
Could you replace this with either the `pprint` module or `json.dumps` with an `indent` param set? There doesn't seem to be a need for us to have our own version.